banyand/metadata/schema/utils.go

```diff
@@ -0,0 +1,140 @@
+// Licensed to Apache Software Foundation (ASF) under one or more contributor
```

Comment:
How about naming this file to `checker.go` instead of a meaningless `util`

Reply:
Renamed